src/components/layout/Layout.tsx

```
@@ -1,14 +1,16 @@
-import React, { ReactNode } from "react";
+import React, { ReactNode, useState } from "react";
```

ESLint

[error] 'useState' is defined but never used.

(@typescript-eslint/no-unused-vars)